### ISSUE TITLE
Display changelog of updated plugins

### ISSUE THREAD

**Issue by kwbr** [k-takata/minpac#35]
I wonder if it's possible to list the changes of the updated plugins on update:

```
cbdf10a Fix #573 - Update the README to recommend installing via the built-in package system
0b50ebb Fix #779 - Handle empty output for tslint
235fc90 Fix #308 - Check Dart files with dartanalyzer
aa94d09 Fix #710 - Show hlint suggestions as info items, and include end line and column numbers
5a6ffc2 Add a missing test file
```

**Comment by hauleth** [comment]
Preferably in quickfix or loclist.

**Comment by k-takata** [comment]
I'm wondering what is the best UI for this.
Currently, minpac uses `:echom` to show the results, and it can be seen by `:messages` later. This is actually almost enough for me, and I don't want to pay much cost for creating fancy UI.

**Comment by hauleth** [comment]
@k-takata maybe quickest/loclist?

**Comment by kwbr** [comment]
```:messages``` would be enough for my taste as well.

**Comment by k-takata** [comment]
Fixed by #58.
